### PR TITLE
Add mingw article

### PR DESCRIPTION
**Pull request by l3x61**
How to install MinGW-w64 using MSYS2.

### CODE REVIEW

File: articles/beginner/mingw.md [hunk 1]
```diff
@@ -0,0 +1,14 @@
+<!-- alias mingw, install-mingw -->
+<!-- title -->
+# How to Install MinGW-w64 
+1. Download and run the [MSYS2](https://www.msys2.org/#installation) installer with the default options.
+2. After installation, a terminal for the [UCRT64 environment](https://www.msys2.org/docs/environments/) will launch. 
+3. In the terminal, run: `pacman -S mingw-w64-ucrt-x86_64-gcc mingw-w64-ucrt-x86_64-gdb`.
```
Comment (jeremy-rifkin) [inline articles/beginner/mingw.md:6]: Can we add `make` and `cmake` to this?

File: articles/beginner/mingw.md [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+4. Open the [Start menu](https://support.microsoft.com/en-us/windows/open-the-start-menu-4ed57ad7-ed1f-3cc9-c9e4-f329822f5aeb) and search for `cmd.exe`.
+5. Right-click on **Command Prompt** and select **Run as administrator**.
+6. Run: `setx PATH "%PATH%;C:\msys64\ucrt64\bin;"`.
```
Comment (jeremy-rifkin) [inline articles/beginner/mingw.md:9]: This is great, so many beginners have trouble changing their path

**Review by jeremy-rifkin — APPROVED** [review]
LGTM! Thanks so much for taking the time to write this, it's very clear, concise, and straightforward. One take or leave request for the default installation and then I can merge and deploy.

File: articles/beginner/mingw.md (hunk 1 above)
Comment (l3x61) [reply articles/beginner/mingw.md:6]: Sure! How about we remove `gdb` from the 'How to Install MinGW-w64' step ...
```suggestion
3. In the terminal, run: `pacman -S mingw-w64-ucrt-x86_64-gcc`.
```

... and add a separate 'common packages' list at the bottom?

## Other packages
To install a new package launch the [UCRT64 environment](https://www.msys2.org/docs/environments/) and run `pacman -S <package-name>`
- `mingw-w64-ucrt-x86_64-gdb` - GNU Debugger
- `mingw-w64-ucrt-x86_64-make` - GNU Make
- `mingw-w64-ucrt-x86_64-cmake` - CMake
- `mingw-w64-ucrt-x86_64-binutils` - GNU Binutils

**Review by jeremy-rifkin — APPROVED** [review]
Awesome thanks so much!

**Pull request merged**